release.py

```diff
@@ -438,6 +438,8 @@ def bump(tag: Tag) -> None:
             "Doc/license.rst",
             "PC/pyconfig.h.in",
             "PCbuild/rt.bat",
+            ".github/ISSUE_TEMPLATE/bug.yml",
+            ".github/ISSUE_TEMPLATE/crash.yml",
```

Comment:
Please could you sort this whole list?

Reply:
One could argue that it's currently sorted logically rather than alphabetically and beyond that the only thing out of order is `LICENSE` before `Doc/license.rst` (if you consider `.` to sort to the end).  Happy to sort as you want, but I don't want to pick the wrong sort :)

Reply:
Good point, let's just leave it like this. They'll be popped open into an editor so it's helpful to have similar ones grouped :)